### PR TITLE
Update to 0.1.4

### PR DESCRIPTION
**Pull request by runiq**

### CODE REVIEW

File: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.metainfo.xml [hunk 1]
```diff
@@ -15,6 +15,19 @@
   <url type="bugtracker">https://github.com/mikeoliphant/neural-amp-modeler-lv2/issues</url>
   <url type="homepage">https://github.com/sdatkinson/neural-amp-modeler</url>
   <releases>
+    <release version="0.1.4" date="2024-08-17">
+      <url>https://github.com/mikeoliphant/neural-amp-modeler-lv2/releases/tag/0.1.4</url>
+      <description>
+        <p>
+          Release 0.1.4
+        </p>
```
Comment (hfiguiere) [inline org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.metainfo.xml]:
```suggestion
```

Comment (runiq) [reply org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.metainfo.xml]: I'll do that. Can I remove that paragraph for the other releases in this PR or should I file a separate one?

Edit: Eh, it's a small enough change. Just did it all in this PR.

**Comment by hfiguiere** [comment]
if this works you can merge anytime.

**Comment by runiq** [comment]
> if this works you can merge anytime.

Haven't had time to test it yet, I'm afraid :( Tonight though.

**Pull request merged**